### ISSUE TITLE
Multifox 2.0b5: does not recognize automatically pages with log-in

### ISSUE THREAD

**Issue by Amarok63** [hultmann/multifox#82]
Hi!

By now I found just two pages, but there are probably more. By elance.com and huffingtonpost.es it won't work. It shows shortly "Ignoreme" and then vanishes.

Greetings.

**Comment by hultmann** [comment]
Are these websites still usable, or do you need to disable Multifox to use them?

**Comment by Amarok63** [comment]
No, they are still usable, but Multifox won't create a new account for 
them so there's no chance to use two profiles of the site at the same time.

**Comment by Drugoy** [comment]
The idea I've described <a href="https://github.com/hultmann/multifox/issues/41#issuecomment-17334564">here</a> would cover this case too.
